### PR TITLE
CRI-O: copy openshift images from docker storage to CRI-O storage

### PR DESCRIPTION
**Pull request by runcom**
@stevekuznetsov @kargakis PTAL

Signed-off-by: Antonio Murdaca <runcom@redhat.com>

### CODE REVIEW

**Comment by jeremyeder** [comment]
@chaitanyaenr

File: sjb/config/test_cases/test_branch_origin_extended_conformance_crio.yml [hunk 1]
```diff
@@ -21,6 +21,11 @@ extensions:
         rpm -V "atomic-openshift-utils${last_tag/openshift-ansible/}.git.0.${last_commit}.el7"
     - type: "script"
       title: "install Ansible plugins"
```
Comment (kargakis) [inline sjb/config/test_cases/test_branch_origin_extended_conformance_crio.yml]: install skopeo

File: sjb/config/test_cases/test_branch_origin_extended_conformance_crio.yml [hunk 2]
```diff
@@ -78,6 +83,11 @@ extensions:
         origin_package="$( source hack/lib/init.sh; os::build::rpm::format_nvra )"
         rpm -V "${origin_package}"
     - type: "script"
+      title: "copy openshift images from docker storage to CRI-O storage"
+      timeout: 10800
+      script: |-
+              for i in $(docker images --filter 'reference=openshift/origin-*' --format '{{.Repository}}:{{.Tag}}'); do sudo skopeo copy docker-daemon:$i containers-storage:$i; done
```
Comment (kargakis) [inline sjb/config/test_cases/test_branch_origin_extended_conformance_crio.yml]: reduce the whitespace, also can you break this into separate lines for better readability?
```sh
for i in $(docker images --filter 'reference=openshift/origin-*' --format '{{.Repository}}:{{.Tag}}'); do
 sudo skopeo copy docker-daemon:$i containers-storage:$i
done
```

File: sjb/config/test_cases/test_branch_origin_extended_conformance_crio.yml (hunk 1 above)
Comment (runcom) [reply sjb/config/test_cases/test_branch_origin_extended_conformance_crio.yml]: fixed

**Comment by kargakis** [comment]
You also need to update the generated xml that will be pushed in Jenkins

./sjb/generate.sh

File: sjb/config/test_cases/test_branch_origin_extended_conformance_crio.yml (hunk 2 above)
Comment (runcom) [reply sjb/config/test_cases/test_branch_origin_extended_conformance_crio.yml]: fixed

**Comment by runcom** [comment]
> You also need to update the generated xml that will be pushed in Jenkins
> 
> ./sjb/generate.sh

Done

**Comment by kargakis** [comment]
The steps for testing are documented in https://github.com/openshift/aos-cd-jobs#custom-xml-generator-configuration-under-sjb

**Comment by runcom** [comment]
@kargakis is there any way to trigger CRI-O jobs with something like `/crio`? thanks for instructions and for kicking it for me, appreciated.

**Comment by kargakis** [comment]
Yes, in an Origin PR you should be able to `/test crio`. For example, https://github.com/openshift/origin/pull/16124#issuecomment-328795498.

**Comment by kargakis** [comment]
We can also run the job when certain files change in Origin, eg. kubelet.

**Comment by runcom** [comment]
@kargakis it seems the job you triggered passed the first batch of tests but failed on an unrelated error about etcd

**Comment by runcom** [comment]
If it's a flake might be nice to restart it

**Comment by kargakis** [comment]
Flake, updating the real job and kicking it..

**Comment by kargakis** [comment]
New build running at https://ci.openshift.redhat.com/jenkins/view/All/job/test_branch_origin_extended_conformance_crio/18/

**Pull request merged**